test/common/performance/aggregator/aggregator.go

```diff
@@ -170,34 +171,34 @@ func (ag *Aggregator) Run(ctx context.Context) {
 			publishErrorTimestamps = append(publishErrorTimestamps, timestampSent)
 
 			if qerr := client.Quickstore.AddError(mako.XTime(timestampSent), publishFailureMessage); qerr != nil {
-				log.Printf("ERROR AddError for publish-failure: %v", qerr)
+				log.Printf("ERROR AddError: %v", qerr)
```

Comment:
I think the improvements to error messages should be kept.

Reply:
Uops honestly i didn't saw them! Sorry